api/src/main/java/org/openmrs/api/impl/AdministrationServiceImpl.java

```diff
@@ -75,20 +78,28 @@
  * @see org.openmrs.api.AdministrationService
  * @see org.openmrs.api.context.Context
  */
+@Service("adminService")
 @Transactional
 public class AdministrationServiceImpl extends BaseOpenmrsService implements AdministrationService, GlobalPropertyListener {
 	
 	private static final Logger log = LoggerFactory.getLogger(AdministrationServiceImpl.class);
 
+	@Autowired
 	protected AdministrationDAO dao;
```

Comment:
Have you tested startup time? There's an issue with using xml defined services with Autowired, which affects startup time significantly, see https://openmrs.atlassian.net/browse/TRUNK-6363. I haven't seen it to be the case if you define service with annotation and then autowire xml defined beans, but just wanted to confirm that. Eventually we could also move DAO from xml to annotations, but I'd do that in a separate issue.

Reply:
okay, let me take a look